test/etoTest.json

```diff
@@ -0,0 +1,42 @@
+[
+  {
+    "description": "Badgerys Creek, AU for May 2019",
+    "source": "http://www.bom.gov.au/watl/eto/tables/nsw/badgerys_creek/badgerys_creek-201905.csv",
+    "elevation": 266,
+    "coordinates": [ -33.90, 150.73 ],
+    "startTimestamp": 1556668800,
+    "entries": [
+      {"eto":0.075,"data":{"maxTemp":76.46,"minTemp":55.04,"maxHumidity":100,"minHumidity":58,"windSpeed":2.309,"solarRadiation":10.4}},
```

Comment:
This is interesting. I was getting eto around 2-4mm for London which is a different magnitude than these test results would suggest. I need to take a look at this and see what I am doing wrong.

Reply:
Rookie mistake. inches vs mm. Why cant we all go metric!